content/docs/contributors.md

```diff
@@ -0,0 +1,35 @@
+# Contributors
```

Comment:
Good to have this as a separate page, back link it to the important pages i.e. README and CONTRIBUTING guide pages.